miso-web/src/main/webapp/scripts/project_ajax.js

```diff
@@ -35,6 +35,12 @@ var Project = Project || {
     jQuery('#alias').attr('data-parsley-maxlength', '100');
     jQuery('#alias').attr('data-parsley-pattern', Utils.validation.sanitizeRegex);
 
+    // Short name input field validation
+    jQuery('#shortName').attr('class', 'form-control');
+    jQuery('#shortName').attr('data-parsley-required', 'true');
+    jQuery('#shortName').attr('data-parsley-maxlength', '100');
```

Comment:
Should be limited to 5 characters, or will fail sample alias validation when used by generator